src/keyboard.rs

```diff
@@ -1755,51 +1758,3 @@ bitflags! {
         const RSUPER   = 0b1000_0000;
     }
 }
-
-#[cfg(feature = "serde")]
-mod modifiers_serde {
-    use serde::{Deserialize, Deserializer, Serialize, Serializer};
-
-    use super::ModifiersState;
-
-    #[derive(Default, Serialize, Deserialize)]
-    #[serde(default)]
-    #[serde(rename = "ModifiersState")]
-    pub struct ModifiersStateSerialize {
-        pub shift_key: bool,
-        pub control_key: bool,
-        pub alt_key: bool,
-        pub super_key: bool,
-    }
-
-    impl Serialize for ModifiersState {
-        fn serialize<S>(&self, serializer: S) -> Result<S::Ok, S::Error>
-        where
-            S: Serializer,
-        {
-            let s = ModifiersStateSerialize {
-                shift_key: self.shift_key(),
-                control_key: self.control_key(),
-                alt_key: self.alt_key(),
-                super_key: self.super_key(),
-            };
-            s.serialize(serializer)
-        }
-    }
-
-    impl<'de> Deserialize<'de> for ModifiersState {
-        fn deserialize<D>(deserializer: D) -> Result<Self, D::Error>
-        where
-            D: Deserializer<'de>,
-        {
-            let ModifiersStateSerialize { shift_key, control_key, alt_key, super_key } =
-                ModifiersStateSerialize::deserialize(deserializer)?;
-            let mut m = ModifiersState::empty();
-            m.set(ModifiersState::SHIFT, shift_key);
-            m.set(ModifiersState::CONTROL, control_key);
-            m.set(ModifiersState::ALT, alt_key);
-            m.set(ModifiersState::SUPER, super_key);
-            Ok(m)
-        }
-    }
-}
```

Comment:
That's a breaking change.

Reply:
We could probably just use the bitflags here, but we should document properly that it's a breaking change.

Reply:
Done.